plugins/draw-tools.js

```diff
@@ -728,6 +728,47 @@ window.plugin.drawTools.initMPE = function(){
   });
 }
 
+//
+// Basic getters for filter to use
+//
+
+window.plugin.drawTools.getMarkers = function () {
+  var markers = [];
+  for (var i in plugin.drawTools.drawnItems._layers) {
+    var layer = plugin.drawTools.drawnItems._layers[i];
+    if (layer instanceof L.Marker) {
+      markers.push(layer.getLatLng());
+    }
+  }
+  return markers;
+};
+
+window.plugin.drawTools.getPolylines = function () {
+  var polylines = [];
+  for (var i in plugin.drawTools.drawnItems._layers) {
+    var layer = plugin.drawTools.drawnItems._layers[i];
+    if (layer instanceof L.GeodesicPolyline) {
+      polylines.push(layer.getLatLngs());
+    }
+  }
+  return polylines;
+};
+
+window.plugin.drawTools.getPolygons = function () {
+  var polygons = [];
+  for (var i in plugin.drawTools.drawnItems._layers) {
+    var layer = plugin.drawTools.drawnItems._layers[i];
+    if (layer instanceof L.GeodesicPolygon) {
+      polygons.push(layer.getLatLngs());
```

Comment:
Need to investigate if we can access projected data directly instead.

Reply:
there is no public method/property, it also depends on the zoom and the renderer afaik

Reply:
Right, but we can use private (I hope).
It definitely depends on zoom, but the result (in/out) is the same.

Reply:
the `._rings` may be the one